README.md

```diff
@@ -244,6 +244,10 @@ These are the individual test-suites (all of these can be suffixed with `-all` t
 -   `make test-proof`: Proofs from the [Verified Smart Contracts].
 -   `make test-interactive`: Tests of the `kevm` command.
 
+You can customize the run by giving extra arguments to make via `PYTEST_ARGS+='-n 12 -vv'`.
```

Comment:
I'd formulate this first in general terms, then with particular examples:

~~~suggestion
All these targets call `pytest` under the hood. You can pass additional arguments to the call by appending them to variable `PYTEST_ARGS`. E.g. run

```
make test-vm PYTEST_ARGS+=-vv
```

to execute VMTests with increased verbosity, and

```
make test-vm PYTEST_ARGS+=-n0
```

to execute them on a single worker.
~~~